components/scream/src/physics/rrtmgp/CMakeLists.txt

```diff
@@ -38,6 +38,9 @@ target_include_directories(rrtmgp PUBLIC
     ${EAM_RRTMGP_DIR}/external/cpp/rrtmgp/kernels
 )
 
+target_compile_definitions(rrtmgp PRIVATE KOKKOS_DEPENDENCE)
```

Comment:
Would you mind adding a couple of lines above this to explain what compile/link option is? I'm sure people will be wondering what it is, and why other places don't have it...

Reply:
@bartgol , yes, when the AT finishes.